seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -6757,7 +6757,7 @@
         "If <code>prop</code> is <code>\"tracks\"</code> but the album doesn't have a <code>\"tracks\"</code> property, create an empty array before adding the new value to the album's corresponding property.",
         "If <code>prop</code> is <code>\"tracks\"</code> and <code>value</code> isn't empty (<code>\"\"</code>), push the <code>value</code> onto the end of the album's existing <code>tracks</code> array.",
         "If <code>value</code> is empty (<code>\"\"</code>), delete the given <code>prop</code> property from the album.",
-        "<strong>Hints</strong><br>Use <code>bracket notation</code> when <a href=\"accessing-objects-properties-with-variables\" target=\"_blank\">accessing object properties with variables</a>.",
+        "<strong>Hints</strong><br>Use <code>bracket notation</code> when <a href=\"javascript-algorithms-and-data-structures/basic-javascript/accessing-objects-properties-with-variables\" target=\"_blank\">accessing object properties with variables</a>.",
```

Comment:
Could you change this to the following:

`javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables`